docs/src/user_docs/assembly/field_operations.md

```diff
@@ -36,7 +36,7 @@ For instructions where one or more operands can be provided as immediate paramet
 | Instruction      | Stack_input | Stack_output   | Notes                         |
 | ---------------- | ----------- | -------------- | ----------------------------- |
 | eq <br> - *(1 cycle)*  <br> eq.*b* <br> - *(1-2 cycles)*   | [b, a, ...] | [c, ...]       | $c \leftarrow \begin{cases} 1, & \text{if}\ a=b \\ 0, & \text{otherwise}\ \end{cases}$ |
-| neq <br> - *(1 cycle)*  <br> neq.*b* <br> - *(1-2 cycles)*  | [b, a, ...] | [c, ...]       | $c \leftarrow \begin{cases} 1, & \text{if}\ a \ne b \\ 0, & \text{otherwise}\ \end{cases}$ |
+| neq <br> - *(1 cycle)*  <br> neq.*b* <br> - *(2-3 cycles)*  | [b, a, ...] | [c, ...]       | $c \leftarrow \begin{cases} 1, & \text{if}\ a \ne b \\ 0, & \text{otherwise}\ \end{cases}$ |
```

Comment:
`neq` cycle count should be set to 2 (only the `neq.b` cycle count seems to have been updated).

Reply:
ops, missed that, thanks!